azext_iot/common/certops.py

```diff
@@ -102,6 +102,52 @@ def create_self_signed_certificate(
     return result
 
 
+def isBase64(content: str) -> bool:
+    """
+    Checks if certificant content should be valid base64 string without prefix and suffix
+
+    Args:
+        content (str): certificant content without prefix and suffix.
+
+    Returns:
+        isBase64 (bool): returns where the certificant content is valid base64 value.
+    """
+    try:
+        sb_bytes = bytes(content, "ascii")
+        base64.b64decode(sb_bytes)
+    except Exception:
+        return False
+    return True
+
+
+def getCertificateFormatValidation(certificate: str) -> str:
```

Comment:
naming conventions: getCertificateFormatValidation -> `get_certificate_format_validation`

Reply:
https://realpython.com/python-pep8/#naming-styles - if you are curious, tdlr: most things are underscore